src/jdk.jartool/share/classes/sun/tools/jar/resources/jar_de.properties

```diff
@@ -104,14 +105,14 @@ main.usage.summary.try=Verwenden Sie "jar --help", um weitere Informationen anzu
 main.help.opt.main=\ Hauptvorgangsmodus:\n
 main.help.opt.main.create=\  -c, --create               Erstellt das Archiv. Wenn der mit\n                             -f oder --file angegebene Archivdateiname einen Pfad enth\u00E4lt, werden auch fehlende\n                             \u00FCbergeordnete Verzeichnisse erstellt
-main.help.opt.main.generate-index=\  -i, --generate-index=FILE  Indexinformationen f\u00FCr die angegebenen JAR-\n                             Archive generieren
+main.help.opt.main.generate-index=\  -i, --generate-index=FILE  Generiert Indexinformationen f\u00FCr die angegebenen\n                             JAR-Archive. Diese Option ist veraltet und wird m\u00F6glicherweise in \n                             einem zuk\u00FCnftigen Release entfernt.
 main.help.opt.main.list=\  -t, --list                 Das Inhaltsverzeichnis f\u00FCr das Archiv auflisten
 main.help.opt.main.update=\  -u, --update               Ein vorhandenes JAR-Archiv aktualisieren
 main.help.opt.main.extract=\  -x, --extract              Benannte (oder alle) Dateien aus dem Archiv extrahieren
-main.help.opt.main.describe-module=\  -d, --describe-module      Gibt die Modulbeschreibung oder den automatischen Modulnamen aus
+main.help.opt.main.describe-module=\  -d, --describe-module      Gibt den Moduldeskriptor oder automatischen Modulnamen aus
 main.help.opt.main.validate=\      --validate             Validiert den Inhalt des JAR-Archivs. Diese Option\n                             validiert, dass die von einem Multi-Release-JAR-Archiv\n                             exportierte API \u00FCber die verschiedenen Releaseversionen\n                             hinweg konsistent ist.
 main.help.opt.any=\ In jedem Modus g\u00FCltige Vorgangsmodifikatoren:\n\n  -C DIR                     Zum angegebenen Verzeichnis wechseln und die folgende\n                             Datei aufnehmen
-main.help.opt.any.file=\  -f, --file=FILE            Der Name der Archivdatei. Wenn Sie dies auslassen, wird entweder stdin oder\n                             stdout verwendet, je nach Vorgang\n      --release VERSION      Speichert alle der folgenden Dateien in einem versionierten Verzeichnis\n                             der JAR-Datei (d.h. META-INF/versions/VERSION/)
+main.help.opt.any.file=\  -f, --file=FILE            Der Name der Archivdatei. Wenn Sie dies lauslassen, wird entweder stdin oder\n                             stdout verwendet, je nach Vorgang\n      --release VERSION      Speichert alle der folgenden Dateien in einem versionierten Verzeichnis\n                             der JAR-Datei (d.h. META-INF/versions/VERSION/)
```

Comment:
This change looks suspicious to me. The English word used is “omitted”. “Auslassen” is to “Leave out”, while “lauslassen” is to “let go”. A native German speaker should confirm, I have only high school level German..